src/components/Catalog.jsx

```diff
@@ -21,24 +21,24 @@ const Catalog = ({ data, pcData }) => {
                 "flex",
                 "flex-col",
                 "mx-auto",
-                pokemonType[0] == "grass" && "bg-[#78C850]",
-                pokemonType[0] == "bug" && "bg-[#A8B820]",
-                pokemonType[0] == "fire" && "bg-[#F08030]",
-                pokemonType[0] == "water" && "bg-[#6890F0]",
-                pokemonType[0] == "normal" && "bg-[#A8A878]",
-                pokemonType[0] == "poison" && "bg-[#A040A0]",
-                pokemonType[0] == "electric" && "bg-[#F8D030]",
-                pokemonType[0] == "ice" && "bg-[#98D8D8]",
-                pokemonType[0] == "fighting" && "bg-[#C03028]",
-                pokemonType[0] == "ground" && "bg-[#E0C068]",
-                pokemonType[0] == "flying" && "bg-[#A890F0]",
-                pokemonType[0] == "psychic" && "bg-[#F85888]",
-                pokemonType[0] == "rock" && "bg-[#B8A038]",
-                pokemonType[0] == "ghost" && "bg-[#705898]",
-                pokemonType[0] == "dark" && "bg-[#705848]",
-                pokemonType[0] == "dragon" && "bg-[#B8B8D0]",
-                pokemonType[0] == "steel" && "bg-[#B8B8D0]",
-                pokemonType[0] == "fairy" && "bg-[#F0B6BC]"
+                pokemonType[0] === "grass" && "bg-[#78C850]",
```

Comment:
Seems kinda messy with this being all here. Let's do a util function instead that accepts pokemonType.
Create a separate file then export it from there. Function should return the class name.

Reply:
> Seems kinda messy with this being all here. Let's do a util function instead that accepts pokemonType. Create a separate file then export it from there. Function should return the class name.

This is done. Please check latest commit for branch task/5-separate-api-calls